docs/css/avatar.css

```diff
@@ -0,0 +1,42 @@
+.bat-avatar-xl {
+    height: 12rem;
+    width: 12rem;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+
+.bat-avatar-large {
+    height: 10rem;
+    width: 10rem;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+.bat-avatar-md {
+    height: 6rem;
+    width: 6em;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+.bat-avatar-sm {
+    height: 3rem;
+    width: 3em;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+.bat-avatar-xs {
+    height: 1.5rem;
+    width: 1.5em;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+
```

Comment:
Avoid giving multiple newline for consistency, use code formatter for formatting.